src/elements/content-uploader/DroppableContent.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+import ItemList from './ItemList';
+import UploadState from './UploadState';
+
+import makeDroppable from '../common/droppable';
+import type { UploadFile, UploadFileWithAPIOptions, UploadItem } from '../../common/types/upload';
+import type { DOMStringList, View } from '../../common/types/core';
+
+import './DroppableContent.scss';
+
+export interface DroppableContentProps {
+    addDataTransferItemsToUploadQueue: (droppedItems: DataTransfer, itemUpdateCallback?: Function) => void;
+    addFiles: (
+        files?: Array<UploadFileWithAPIOptions | UploadFile>,
+        itemUpdateCallback?: Function,
+        isRelativePathIgnored?: boolean,
+    ) => void;
```

Comment:
you should write the types of these functions based on how it's used/passed within this component.

in this file, the `addDataTransferItemsToUploadQueue` and `addFiles` functions only pass one parameter